install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.agent (not .Values.preflight.enabled) }}
+{{- if and .Values.agent (not .Values.preflight.enabled) .Values.rbac.create }}
```

Comment:
Do you actually need to disable RBAC? Otherwise I'd say let's just remove the feature to disable rbac. It clearly hasn't been working for a while now and no one complained, so I say let's just remove the Helm value instead.

Reply:
I need multiple instances of Cilium chart in a single cluster - there are multiple OS clusters each having Cilium control plane in a special K8S cluster (which is not part of the cluster mesh). Cluster role and binding are cluster-wide resources and can belong to one chart only.